src/assets/scripts/airport/AirportController.js

```diff
@@ -200,8 +200,8 @@ export default class AirportController {
         const runwaySecondaryEndIndex = 1;
         const runways = this.airport_get().runways;
         for (let runwayPair = 0; runwayPair < runways.length; runwayPair++) {
-            runways[runwayPair][runwayPrimaryEndIndex].removeQueue(aircraft, true);
-            runways[runwayPair][runwaySecondaryEndIndex].removeQueue(aircraft, true);
```

Comment:
I don't remember now, but I remember that when looking at it, I determined that it is no longer a factor. :smile: